nitrocli/Cargo.toml

```diff
@@ -58,4 +63,7 @@ path = "../pkg-config"
 "hidapi-sys:0.1.4" = { path = "../hidapi-sys" }
 "libc:0.2.45" = { path = "../libc" }
 "libhid:0.4.1" = { path = "../hid" }
+"libnitrokey:0.2.1" = { path = "../nitrokey" }
+"nitrokey-sys:3.4.1" = { path = "../nitrokey-sys" }
 "pkg-config:0.3.14" = { path = "../pkg-config" }
+"rand:0.4.3" = { path = "../rand" }
```

Comment:
Out of curiosity, do you any plans for making the `nitrokey` crate compatible with more recent versions of `rand`? It appears they have a [comprehensive documentation](https://rust-random.github.io/book/update-0.5.html) for the upgrade, I just haven't checked how much of the `rand` code you use.

Reply:
Yes, I’m planning to do that soon.  It should not be that hard as `nitrokey` only uses `rand` to generate a temporary password.